recipes/libcurl/all/conanfile.py

```diff
@@ -230,7 +230,7 @@ def _get_configure_command_args(self):
             "--with-libmetalink={}".format(yes_no(self.options.with_libmetalink)),
             "--with-libpsl={}".format(yes_no(self.options.with_libpsl)),
             "--with-schannel={}".format(yes_no(self.options.with_ssl == "schannel")),
-            "--with-darwinssl={}".format(yes_no(self.options.with_ssl == "darwinssl")),
+            "--with-secure-transport={}".format(yes_no(self.options.with_ssl == "darwinssl")),
```

Comment:
With `schannel` would this disable TLS?

Reply:
https://github.com/curl/curl/blob/47e169e7a46dd0932ede5225768aa666730820f1/configure.ac#L196